recipes/jupyterlab_launcher/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.1.4" %}
+
+package:
+  name: jupyterlab_launcher
+  version: {{ version }}
+
+source:
+  fn: jupyterlab_launcher-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/j/jupyterlab_launcher/jupyterlab_launcher-{{ version }}.tar.gz
+  sha256: feab645edd2a06f3a40a04b45e6dd3001101f094fe11ff81f649d589704fdb54
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - notebook
+
+  run:
+    - python
+    - notebook
+
+test:
+  # Python imports
+  imports:
+    - jupyterlab_launcher
+
+about:
+  home: http://github.com/jupyterlab/jupyterlab_launcher
+  license: BSD 3-Clause
```

Comment:
So actually all you need to add here is `license_file: LICENSE`. This will pull the license file from the source.